.tool-versions

```diff
@@ -1 +1 @@
-nodejs 14.15.5
+nodejs 14.5.0
```

Comment:
Did we want to downgrade?

Reply:
Not with great conviction. We were initially planning to go with node 15 but it doesn't have long term support, so we ended up defaulting to the version Friendly was using since we know that one works. I imagine 14.15.5 will not be noticeably different though. If you think it's worth upgrading I'll make the change.

Reply:
Went ahead and updated to 14.15.5 after experimenting a bit and finding no issues.